### PR TITLE
fix(tracking): allow updating billed reports

### PR DESCRIPTION
**Pull request by trowik**
updating billed reports should be allowed, as there are projects, that are billed in advance

### CODE REVIEW

**Review by winged — APPROVED** [review]
LGTM

**Review by anehx — APPROVED** [review]
Nice work!

**Comment by trowik** [comment]
Complexity is too high :unamused: 
Any ideas on how to lower it? Or should I just increase the max-complexity for flake8?

**Comment by anehx** [comment]
@trowik use the `validate_billed` method for validating this. You can use `validate_{field_name}` for all validations that are specific to one field only.

**Comment by trowik** [comment]
re-requested review for the validate_billed change

File: timed/tracking/serializers.py [hunk 1]
```diff
@@ -117,6 +117,15 @@ def validate_duration(self, value):
         """Only owner is allowed to change duration."""
         return self._validate_owner_only(value, "duration")
 
+    def validate_billed(self, value):
+        """Only accountatns may bill reports."""
+        if self.instance is not None:
+            user = self.context["request"].user
+            if not user.is_accountant and (self.instance.billed != value):
```
Comment (anehx) [inline timed/tracking/serializers.py:124]:
```suggestion
            if not self.context["request"].user.is_accountant and (self.instance.billed != value):
```

**Review by anehx — CHANGES_REQUESTED** [review]
Otherwise LGTM :+1:

**Review by anehx — APPROVED** [review]

**Pull request merged**